front/app/containers/Admin/projects/project/projectHeader/PublicationButtons/ReviewFlow/index.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+import { Button, Tooltip, Box } from '@citizenlab/cl2-component-library';
+
+import useApproveProjectReview from 'api/project_reviews/useApproveProject';
+import useProjectReview from 'api/project_reviews/useProjectReview';
+import { IProjectData } from 'api/projects/types';
+
+import { useIntl } from 'utils/cl-intl';
+import { usePermission } from 'utils/permissions';
+
+import messages from '../messages';
+
+import ReviewRequestButton from './ReviewRequestButton';
+
+interface Props {
+  project: IProjectData;
+}
+
+const ReviewFlow = ({ project }: Props) => {
+  const canReview = usePermission({
+    item: project,
+    action: 'review',
+  });
+  const { data: projectReview, isLoading: isProjectReviewLoading } =
+    useProjectReview(project.id);
+  const {
+    mutate: approveProjectReview,
+    isLoading: isApprovingProjectReviewLoading,
+  } = useApproveProjectReview();
+  const { formatMessage } = useIntl();
+
+  const approvalPending = projectReview?.data.attributes.state === 'pending';
+  const approvalGranted = projectReview?.data.attributes.state === 'approved';
+
+  const showProjectApprovalButton = approvalPending && canReview;
+  const showReviewRequestButton = !canReview && !approvalGranted;
```

Comment:
Took out the feature flag check here since we know that is true now if this component renders, the rest of the file is unchanged.